corehq/pillows/mappings/user_mapping.py

```diff
@@ -216,7 +216,10 @@
                 },
                 "timezone": {
                     "type": "string"
-                }
+                },
+                'assigned_location_ids': {
+                    'type': 'string'
+                },
```

Comment:
This is inside `domain_memberships`, which applies to web users, but there's also a `domain_membership` (singular) blob above this which is otherwise identical.  I'd apply the changes there too for consistency.

Reply:
@esoergel Why do we have both `domain_membership` and `domain_memberships`?

Reply:
`CommCareUser`s have a `domain_membership` field and `WebUser`s have a `domain_memberships` field.  I do think `commcare_user.domain_membership.location_id` is redundant with `commcare_user.location_id`, but we do store it in both places

https://github.com/dimagi/commcare-hq/blob/479e5795245a01d38283d50dc079c79c31986b46/corehq/apps/users/models.py#L2048-L2049